lexers/c.lua

```diff
@@ -22,7 +23,8 @@ lex:add_rule('function', (builtin_func + method + func) * #(lexer.space^0 * '(')
 -- Constants.
 lex:add_rule('constants', lex:tag(lexer.CONSTANT_BUILTIN,
 	-(B('.') + B('->')) * lex:word_match(lexer.CONSTANT_BUILTIN)))
-
+lex:add_rule('linux-constants', lex:tag(lexer.CONSTANT_BUILTIN .. '.linux',
```

Comment:
I would prefer not to have domain-specific things in a standard C lexer. See comment here: https://github.com/orbitalquark/scintillua/discussions/124#discussioncomment-11643726

Sorry about that.